godot-bindings/src/lib.rs

```diff
@@ -161,3 +161,25 @@ pub fn emit_godot_version_cfg() {
         println!(r#"cargo:rustc-cfg=gdextension_exact_api="{major}.{minor}""#);
     }
 }
+
+// Function for safely removal of build directory. Workaround for errors happening during CI builds:
+// https://github.com/godot-rust/gdext/issues/616
+pub fn remove_dir_all_safe(path: &std::path::Path) {
+    let mut retry_count = 0;
+
+    while retry_count <= 5 && path.exists() {
+        match std::fs::remove_dir_all(path) {
+            Ok(_) => break,
+            Err(err) => {
+                if retry_count >= 5 {
+                    panic!(
+                        "cannot remove directory: {path_display} after 5 tries with error: {err}",
+                        path_display = path.display()
+                    )
+                }
```

Comment:
You can use `assert_ne!` here -- it's impossible that this is ever larger than 5.

Reply:
Shoot, botched it a little. Never gone up to 5 retries yet, but I will change the `while` loop condition for this to happen (better error message)

Reply:
Tested after changing the while loop condition, all green still

Reply:
Still, the `if retry_count >= 5 { panic!(...) }` can be simplified using `assert_ne!(...)` 🙂 
```suggestion
                assert_ne!(
                    retry_count, 5,
                    "cannot remove directory: {path} after 5 tries with error: {err}",
                    path = path.display()
                )
```